client/flutter/lib/components/emojiHeader.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+class EmojiHeader extends StatelessWidget {
+  EmojiHeader(this.emoji);
+
+  final String emoji;
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Text(
+        this.emoji,
```

Comment:
`this` is not needed

Reply:
Its good practice to be clear

Reply:
There's actually a linter rule suggesting `this` shouldn't be used
https://dart-lang.github.io/linter/lints/unnecessary_this.html